flink-simple-tutorial/README.md

```diff
@@ -87,9 +87,9 @@ The `HeapMetrics` class has a few key properties:
 2. It has a public empty constructor
 3. All fields are public non-final
 
-With these properties, the class is made to be efficiently serializable by the Flink type system.
+These classes are called POJOs (Plain Old Java Objects) in the Flink community. They are supported out of the box by Flink for serialization. As Flink is a distributed system transferring data between machines is a fundamental functionality and a potential performance bottleneck. 
 
-These classes are called POJOs (Plain Old Java Objects) in the Flink community. It is possible to structure the class differently by keeping the same serialization properties. For the exact rules, see the [upstream documentation](https://ci.apache.org/projects/flink/flink-docs-stable/dev/types_serialization.html#rules-for-pojo-types). 
+Consequently the community offers efficient serialization support for a range of commonly used types, while preserving flexibility for the user to choose to plug in their own serialization if needed, however we recommend to stick to the battle tested defaults. For the exact rules, see the [upstream documentation](https://ci.apache.org/projects/flink/flink-docs-stable/dev/types_serialization.html#rules-for-pojo-types). 
```

Comment:
Consequently, the community offers efficient serialization support for a range of commonly used types, while preserving flexibility. The user can choose to plug in their own serialization if needed, however, we recommend to stick to the battle tested defaults.